src/elements/content-sidebar/activity-feed/annotations/AnnotationActivity.js

```diff
@@ -85,7 +121,28 @@ const AnnotationActivity = ({
                     <div>
                         <ActivityTimestamp date={createdAtTimestamp} />
                     </div>
-                    <ActivityMessage id={id} tagged_message={message} getUserProfileUrl={getUserProfileUrl} />
+                    {isEditing ? (
+                        <CommentForm
+                            isDisabled={isDisabled}
+                            className={classNames('bcs-Comment-editor', {
+                                'bcs-is-disabled': isDisabled,
+                            })}
+                            updateComment={commentFormSubmitHandler}
+                            isOpen={isInputOpen}
+                            // $FlowFixMe
```

Comment:
What is this fixing?

Reply:
```
Cannot create CommentForm element because
undefined [1] is incompatible with UserMini [2]
in property user.
```

I think since on initial load, it's possible for currentUser to not be defined but later on it does become defined.

Reply:
Can we fix it by checking the existence of `currentUser` together with `isEditing`, like `isEditing && currentUser`?